### PR TITLE
WEBEU2 - Build Week FE - Josemaria Nriagu 🇳🇬

### PR DESCRIPTION
**Pull request by josenriagu**
_Set up project_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (maximesalomon) [inline README.md]: Please update the Readme with your Technical Design Document

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (maximesalomon) [inline public/manifest.json]: You might want to change this as well!

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+class App extends Component {
+  render() {
+    return (
+      <>
+        <GlobalStyle />
+        {
+          localStorage.getItem('token')
+            ?
+            <Header />
```
Comment (maximesalomon) [inline src/App.js]: localStorage.getItem('token') && <Header />

File: README.md (hunk 1 above)
Comment (josenriagu) [reply README.md]: Okay, I will do that ASAP

File: public/manifest.json (hunk 2 above)
Comment (josenriagu) [reply public/manifest.json]: Sure

**Pull request merged**